functions/completed-checks/index.js

```diff
@@ -1,11 +1,60 @@
 'use strict'
 
-const uuid = require('uuid/v4')
 const moment = require('moment')
+const sqlService = require('less-tedious')
+const uuid = require('uuid/v4')
+const winston = require('winston')
+const { TYPES } = require('tedious')
+const R = require('ramda')
+
+winston.level = 'error'
+const config = require('../config')
+sqlService.initialise(config)
+const { deleteFromPreparedCheckTableStorage, getPromisifiedAzureTableService } = require('../lib/lib')
+
+// SQL server table
+const checkResultTable = '[checkResult]'
+const checkFormAllocationTable = '[checkFormAllocation]'
+const schema = ['mtc_admin']
+const checkStatusTable = '[checkStatus]'
+const checkTable = '[checkFormAllocation]'
 
-module.exports = function (context, completedCheckMessage) {
+// Table Storage
+const azureTableService = getPromisifiedAzureTableService()

```

Comment:
@jon-shipley I still think promisifying the whole library for 2 method calls along with any potential internal dependency in other methods is too much.
The time it takes for the library 'promisification' to complete might be small but I believe it's unnecessary for the minimal usage you have.

Reply:
I am resolving this based on the answer given by Guy on #821